fastapi/encoders.py

```diff
@@ -42,25 +42,6 @@ def _jsonable_encoder(
     custom_encoder: dict,
     sqlalchemy_safe: bool,
 ) -> Any:
-    if isinstance(obj, BaseModel):
```

Comment:
after this https://gist.github.com/podhmo/adebe386eaf298b5fb7a1689f57f027e#file-02benchmark-txt